docs/conf.py

```diff
@@ -363,3 +396,24 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
+
+# sphinx-autoapi configuration
+# See:
+# https://sphinx-autoapi.readthedocs.io/en/latest/config.html
+
+# Paths (relative or absolute) to the source code that you wish to generate
+# your API documentation from.
+autoapi_dirs = [
+    os.path.abspath('../airflow'),
+]
+
+# A directory that has user-defined templates to override our default templates.
+autoapi_template_dir = 'autoapi_templates'
+
+# Keep the AutoAPI generated files on the filesystem after the run.
+# Useful for debugging.
+autoapi_keep_files = False
+
+# Relative path to output the AutoAPI files into. This can also be used to place the generated documentation
+# anywhere in your documentation hierarchy.
+autoapi_root = '_autoapi'
```

Comment:
Could we have the folder/URL path be `/api-reference` (maybe `/python...Something` to distinguish from an HTTP API)  or similar please? AutoAPI feels like an implementation detail.

Reply:
What do you think about _api? the "_" symbol means that the file is generated automatically.

It will be uniformly with other directory
````
https://airflow.readthedocs.io/en/latest/_modules/airflow/models.html#BaseOperator
https://airflow.apache.org/_sources/index.rst.txt
````

Reply:
Sounds good.

Reply:
I updated a PR.